components/cache.js

```diff
@@ -0,0 +1,427 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const async = require('async');
+const redis = require('redis');
+const transactionTypes = require('../helpers/transaction_types');
+const { CACHE } = require('../config/default/constants');
```

Comment:
As mentioned before, I reckon it should be:

`const CONSTANTS = require('./constants');`